src/utils/hendleError/hendleError.ts

```diff
@@ -0,0 +1,14 @@
+import BaseError from './baseError';
+import { HttpStatusCode } from './httpStatusCode';
+
+
+export default class Api404Error extends BaseError {
```

Comment:
It is a great idea to have a special class for HTTP 404 error. It would be even better to have classes for each http Error code with a default message and with the ability to provide a custom message.  Usage will look like that

`throw new HttpBadRequsetError('my custom message')`
`throw new HttpNotFoundError() // case with default meessage`

But it is not necessary, so you can leave it as is

Reply:
it is a really good idea